noxfile.py

```diff
@@ -20,10 +20,22 @@ def test(session):
     htmlcov_output = os.path.join(session.virtualenv.location, 'htmlcov')
     xmlcov_output = os.path.join(session.virtualenv.location, f'coverage-{session.python}.xml')
 
-    session.install('.', 'importlib_metadata')
-    session.install('pytest', 'pytest-cov', 'pytest-mock')
+    session.install('.[test]')
 
-    session.run('pytest', '--cov',
+    session.run('pytest', '--cov', '--cov-config', 'setup.cfg',
                 f'--cov-report=html:{htmlcov_output}',
                 f'--cov-report=xml:{xmlcov_output}',
                 'tests/', *session.posargs)
+
+
+@nox.session(python=['2.7', '3.5', '3.6', '3.7', '3.8', 'pypy2', 'pypy3'])
+def integration(session):
+    htmlcov_output = os.path.join(session.virtualenv.location, 'htmlcov')
```

Comment:
I'm not on board with having integration tests under their own nox env, they should live in the same namespace as the unit ones, just have a flag to run/not run them 🤷